bin/build-manifest.js

```diff
@@ -0,0 +1,248 @@
+/* eslint-env node */
+"use strict";
+
+var _ = require("lodash");
+var execSync = require("child_process").execSync;
+var fg = require("fast-glob").sync;
+var fs = require("fs");
+
+/**
+ * Clean up output to ensure that special terms, acronyms,
+ * et cetera are properly formatted.
+ *
+ * @param {String} str Input string on which to perform find and replace.
+ * @return {String} The input string with replacements applied.
+ */
+var replaceValuesInString = function (str) {
+    var replacementPatterns = {
+        "a 11 y": "a11y",
+        "a 11 Y": "a11y",
+        "Aria": "ARIA",
+        "Fluid get": "fluid.get",
+        "Html 5": "HTML5",
+        "infusion": "Infusion",
+        "io c": "IoC",
+        "Io c": "IoC",
+        "Jq unit": "jqUnit",
+        "js": "JS",
+        "Ui": "UI",
+        "Url": "URL"
+    };
+
+    Object.entries(replacementPatterns).map(function (pattern) {
+        str = str.replace(pattern[0], pattern[1]);
+    });
+    return str;
+};
+
+var normalizeCase = function (str) {
+    str = _.lowerCase(str);
+    return _.upperFirst(str);
+};
+
+/**
+ * Determine the URL of a demo, example, or test based on file path.
+ *
+ * @param {String} path The file path of a demo, example, or test.
+ *
+ * @return {URL} The URL of the demo, example, or test.
+ */
+var getUrl = function (path) {
+    return new URL("https://build-infusion.fluidproject.org/" + path);
```

Comment:
If this runs as part of a netlify deploy, shouldn't this URL point to the site's own structures? Might be a better place for a relative URL, perhaps. If we're not proposing moving the build site itself into this structure, it might be better to migrate this entire logic into the build site.

Reply:
The manifest file will be ingested by the build.fluidproject.org site so it requires full URLs; you make an excellent point about the index.html file though. It should use relative URLs.